lib/src/base-styles/_nx-btn.scss

```diff
@@ -19,7 +19,7 @@
     color: var(--nx-color-#{$variant}-button-text);
   }
 
-  &:focus:not(:active) {
```

Comment:
These `:not` selectors were causing unnecessary specificity that was getting in the way so I refactored it a bit.  You can see by the fact that no unrelated button visual tests changed that the change did not have unintended consequences.